tests/test_seed.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+
+import unittest
+from monero.seed import Seed, get_checksum
+from exceptions import ValueError
```

Comment:
This seems to fail on Python 3 (at least mine). `ValueError` is a builtin, it doesn't need to be imported. I think this line can be deleted.